test/kubemark/start-kubemark.sh

```diff
@@ -314,8 +346,16 @@ current-context: kubemark-context")
   sed -i'' -e "s/{{EVENTER_MEM}}/${eventer_mem}/g" "${RESOURCE_DIRECTORY}/addons/heapster.json"
   "${KUBECTL}" create -f "${RESOURCE_DIRECTORY}/addons" --namespace="kubemark"
 
+  # When using Cluster Autoscaler, always start with one replica.
+  # NUM_NODES is the maximum number of nodes in the cluster and is used to
+  # calculate the resources for master and addons to be able to handle it.
+  if [[ "${ENABLE_KUBEMARK_CLUSTER_AUTOSCALER}" == "true" ]]; then
```

Comment:
Instead of three `if [[ "${ENABLE_KUBEMARK_CLUSTER_AUTOSCALER}" == "true" ]]` statements in this function, could you just use one?

Reply:
Done.